packages/grid/src/vaadin-grid-column.d.ts

```diff
@@ -121,6 +122,18 @@ declare class GridColumn<TItem = GridDefaultItem> extends HTMLElement {
    */
   path: string | null | undefined;
 
+  /**
+   * When set to true, the row is rendered as a header row, this is an accessibility feature.
```

Comment:
I used chatGPT, I used Github CoPilot, none of them was providing good enough.
Now I also gave context to ChatGPT (Acceptence Criteria) and asked to formulate better, I still dislike the result.

This was what it gave after the AR context:
```
/**
* When set to true, the column is marked as containing row-header cells, providing an accessibility feature.
* The combination of the header cells in this column will be announced by the screen reader,
* identifying the current row as uniquely as possible.
* Cells in these columns are rendered as <th scope="row"> instead of the normal <td role="gridcell">.
* @attr {boolean} row-header
* @type {boolean}
*/
```

I would probably rewrite it like this?
What do you think about this one?
/**
* When set to true, the column will be arow-header cells, providing an accessibility feature.
* The header cells in this column will be announced by the screen reader,
* identifying the current row as uniquely as possible.
* Cells in these columns are rendered as <th scope="row"> instead of the normal <td role="gridcell">.
* @attr {boolean} row-header
* @type {boolean}
*/